Source/Turbo.swift

```diff
@@ -0,0 +1,63 @@
+import WebKit
+
+public enum Turbo {
+    public static var config = TurboConfig()
+}
+
+public class TurboConfig {
+    public typealias WebViewBlock = (_ configuration: WKWebViewConfiguration) -> WKWebView
+
+    /// Override to set a custom user agent.
+    /// - Important: Include "Turbo Native" to use `turbo_native_app?` on your Rails server.
+    public var userAgent = "Turbo Native iOS"
```

Comment:
Does this get used anywhere? I don't see it referenced anywhere in this repo. It's a bit confusing becuase the demo does this:

https://github.com/hotwired/turbo-ios/blob/3ce4f977e1dc8eb5b2eeefb81a07bc344e230584/Demo/SceneController.swift#L16

*and* this:

https://github.com/hotwired/turbo-ios/blob/3ce4f977e1dc8eb5b2eeefb81a07bc344e230584/Demo/Web/WKWebViewConfiguration%2BApp.swift#L16

As far as I can tell only the latter does anything.